package.json

```diff
@@ -497,12 +497,32 @@
           "default": "\\"
         },
         "vim.searchHighlightColor": {
-          "type": "string",
-          "description": "Color of the search highlight."
+          "type": [
+            "string",
+            "undefined"
+          ],
+          "description": "Background color of the other search matches. The color must not be opaque so as not to hide underlying decorations. Uses `editor.findMatchHighlightColor` from current theme if undefined."
```

Comment:
Here and below, you can use `#editor.findMatchHighlightColor#` to make a link to the setting